src/widgets/FramelessEmbedWindow.cpp

```diff
@@ -15,6 +15,24 @@
 
 namespace chatterino {
 
+namespace {
+#ifdef USEWINSDK
+    void sendToStreamView(void *handle, QJsonDocument doc)
+    {
+        auto json = doc.toJson();
+        json.append('\0');
+
+        COPYDATASTRUCT cds;
+        cds.cbData = static_cast<DWORD>(json.size());
+        cds.lpData = json.data();
+
+        ::SendMessage((HWND)handle, WM_COPYDATA,
+                      reinterpret_cast<WPARAM>(handle),
```

Comment:
This should be handle of the window that is passing the data, as per winapi docs

Reply:
Good catch